src/Utils/utils.ts

```diff
@@ -415,9 +414,7 @@ export const formatPatientAge = (
   // Skip representing as no. of months/days if we don't know the date of birth
   // since it would anyways be inaccurate.
   if (!obj.date_of_birth) {
-    return abbreviated
-      ? `Born ${obj.year_of_birth}`
-      : `Born on ${obj.year_of_birth}`;
+    return abbreviated ? `${obj.age} Y` : `Born on ${obj.year_of_birth}`;
```

Comment:
but if `year_of_birth` is undefined, and `abbreviated` is set to false, this would still have the same issue right?

instead let's check if `age` is null or not and show age if available, else fallback to year of birth

Reply:
![image](https://github.com/user-attachments/assets/7bb27d7b-2b80-402f-9987-22537c8471bd)
Is it correct?

Reply:
no, let's keep old year of birth if age is not present. if backend gives us age, let's show age directly.

```suggestion
    if (age != null) return `${obj.age} Y`;
    return abbreviated ? `Born ${obj.year_of_birth}` : `Born on ${obj.year_of_birth}`;
```

Reply:
![image](https://github.com/user-attachments/assets/838ab6e0-1c80-4084-abd3-408e83f472b0)

Is it what you meant?

Reply:
yup!